scripts/generate

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env node
+
+/**
+ * Copyright 2017, Google, Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const childProcess = require('child_process');
+const fs = require(`fs`);
+const path = require(`path`);
+
+const PROJECT_ROOT = path.join(__dirname, `..`);
+
+// Dedupe package.json dependencies
+// WARNING: This will fail if two different versions of the same package are required.
```

Comment:
Actually will remove, it's leftover because some of this was copied from your unify script.